generated/nirfmxwlan/nirfmxwlan.proto

```diff
@@ -31,27 +590,3421 @@ enum Boolean {
+    int32 frequency_error_estimation_method_raw = 4;
+  }
+}
+
+message OFDMModAccCfgFrequencyErrorEstimationMethodResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccCfgMeasurementLengthRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  int32 measurement_offset = 3;
+  int32 maximum_measurement_length = 4;
+}
+
+message OFDMModAccCfgMeasurementLengthResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccCfgMeasurementModeRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof measurement_mode_enum {
+    OfdmModAccMeasurementMode measurement_mode = 3;
+    int32 measurement_mode_raw = 4;
+  }
+}
+
+message OFDMModAccCfgMeasurementModeResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccCfgNoiseCompensationEnabledRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof noise_compensation_enabled_enum {
+    OfdmModAccNoiseCompensationEnabled noise_compensation_enabled = 3;
+    int32 noise_compensation_enabled_raw = 4;
+  }
+}
+
+message OFDMModAccCfgNoiseCompensationEnabledResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccCfgOptimizeDynamicRangeForEVMRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof optimize_dynamic_range_for_evm_enabled_enum {
+    OfdmModAccOptimizeDynamicRangeForEvmEnabled optimize_dynamic_range_for_evm_enabled = 3;
+    int32 optimize_dynamic_range_for_evm_enabled_raw = 4;
+  }
+  double optimize_dynamic_range_for_evm_margin = 5;
+}
+
+message OFDMModAccCfgOptimizeDynamicRangeForEVMResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccCfgPhaseTrackingEnabledRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof phase_tracking_enabled_enum {
+    OfdmModAccPhaseTrackingEnabled phase_tracking_enabled = 3;
+    int32 phase_tracking_enabled_raw = 4;
+  }
+}
+
+message OFDMModAccCfgPhaseTrackingEnabledResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccCfgSymbolClockErrorCorrectionEnabledRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  oneof symbol_clock_error_correction_enabled_enum {
+    OfdmModAccSymbolClockErrorCorrectionEnabled symbol_clock_error_correction_enabled = 3;
+    int32 symbol_clock_error_correction_enabled_raw = 4;
+  }
+}
+
+message OFDMModAccCfgSymbolClockErrorCorrectionEnabledResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccClearNoiseCalibrationDatabaseRequest {
+  nidevice_grpc.Session instrument = 1;
+}
+
+message OFDMModAccClearNoiseCalibrationDatabaseResponse {
+  int32 status = 1;
+}
+
+message OFDMModAccFetchChainDataRMSEVMPerSymbolMeanTraceRequest {
+  nidevice_grpc.Session instrument = 1;
+  string selector_string = 2;
+  double timeout = 3;
+}
+
+message OFDMModAccFetchChainDataRMSEVMPerSymbolMeanTraceResponse {
+  int32 status = 1;
+  double x0 = 2;
+  double dx = 3;
+  repeated float chain_data_rmsevm_per_symbol_mean = 4;
```

Comment:
Note: not part of this change.

`rmsevm` -> `rms_evm`.

Here's an example bug/task on how to track these: https://github.com/ni/grpc-device/issues/465

You don't have to make it a GH issue, it can be an azdo task. But you can use the same laundry list approach.